README.md

```diff
@@ -1,93 +1,186 @@
+    ·
+    <a href="https://github.com/supabase/realtime/issues/new?assignees=&labels=bug&template=1.Bug_report.md">Report Bug</a>
+    <br />
+  </p>
+</p>
 
-Open up three different terminal windows and run the following sequentially:
 
-1. In the first terminal, run `make start.dbs`.
-2. In the second terminal, run `make dev`.
-3. In the third terminal, run `make seed`.
+## Status
 
-### Note
+| Features         |    v1    |    v2    |  Status  |
+|------------------|----------|----------|----------|
+| Postgres Changes |     ✔    |     ✔    |    GA    |
+| Broadcast        |          |     ✔    |   Beta   |  
+| Presence         |          |     ✔    |   Beta   |
 
-- Generate a JWT from `jwt_secret` column on `tenants` table for tenant `dev_tenant`.
+This repository focuses on version 2 but you can still access the previous version's [code](https://github.com/supabase/realtime/tree/single-tenant-main) and [Docker image](https://hub.docker.com/layers/supabase/realtime/v0.25.1/images/sha256-7fc3731782f80e1f5e8b375bce0ebb223229690da1b73997e44dbb31a16fe885?context=explore). For the latest Docker images go to https://hub.docker.com/r/supabase/realtime.
 
-## Motivation
+The codebase is under heavy development and the documentation is constantly evolving. Give it a try and let us know what you think by creating an issue. Watch [releases](https://github.com/supabase/realtime/releases) of this repo to get notified of updates. And give us a star if you like it!
 
-Software is becoming more collaborative. Often the data that we need to share is ephemeral, meaning that it doesn't need to be stored in a database. The goals for ephemeral data are different from stored data:
 
-- Highly Available, Eventually Consistent
-- Changing frequently, not persisted
-- Accessible to other users
-- High number of connections at one time
-- Usually requires presence information - knowing who is online
+## Overview
 
+### What is this?
 
-## Usage
+This is a server built with Elixir using the [Phoenix Framework](https://www.phoenixframework.org) that enables the following functionality:
 
-Install the client library:
+- Broadcast: Send ephemeral messages from client to clients with low latency.
+- Presence: Track and synchronize shared state between clients.
+- Postgres Changes: Listen to Postgres database changes and send them to authorized clients.
 
-```bash
-npm install phoenix
-```
+For a more detailed overview head over to [Realtime guides](https://supabase.com/docs/guides/realtime).
+
+### Does this server guarantee message delivery?
+
+The server does not guarentee that every message will be delivered to your clients so keep that in mind as you're using Realtime. 
+
+
+## Quick start
+
+You can check out the [Multiplayer demo](https://multiplayer.dev) that features Broadcast, Presence and Postgres Changes under the demo directory: https://github.com/supabase/realtime/tree/main/demo.
```

Comment:
`https://github.com/supabase/realtime/tree/main/demo` will work once merged to `main`.